src/executor/Executor.cpp

```diff
@@ -592,9 +592,11 @@ void Executor::drop() {
 Status Executor::finish(Result &&result) {
     if (!FLAGS_enable_lifetime_optimize || node()->outputVarPtr()->lastUser.hasValue()) {
         numRows_ = result.size();
-        ectx_->setResult(node()->outputVar(), std::move(result));
-    } else {
-        VLOG(1) << "Drop variable " << node()->outputVar();
```

Comment:
Don't delete the vlog.

Reply:
I think it's the debug code, need not to put it here when not debugging.